.github/workflows/ci.yml

```diff
@@ -25,3 +26,21 @@ jobs:
 
     - name: Build
       run: make build
+
+  fossa:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write # needed for the Vault authentication
+
+    steps:
+      - name: "Load Secrets from Vault"
+        uses: rancher-eio/read-vault-secrets@main
+        with:
+          secrets: |
+            secret/data/github/repo/${{ github.repository }}/fossa/credentials token | FOSSA
+      - uses: actions/checkout@v3
```

Comment:
I thought for a bit that this belonged to the `load Secrets ...` step instead of being its own.
Needs a bit of separation, and giving it a name would not be bad either.